nomination/admin.py

```diff
@@ -31,8 +31,6 @@ class URLAdmin(admin.ModelAdmin):
 
 class ProjectAdminForm(forms.ModelForm):
     """ Project class to specify how form data is handled in admin """
-    class Meta:
-        model = Project
```

Comment:
yes, because the desired fields weren't explicitly listed. Since we wanted all the fields in the form (why they weren't listed in the first place), the simplest solution was to remove the Meta class, which now has the same effect as not listing the model fields explicitly (including all of them).